src/architecture/style.md

```diff
@@ -41,53 +41,6 @@ This structure is effectively created once per [pipeline][docs-pipeline], in the
 The [properties module][properties-module] is a mako template.
 Its complexity is derived from the code that stores properties, [`cascade` function][properties-cascade-fn] and computation logic of the returned value which is exposed in the main function.
 
-<a name="pseudo-elements"></a>
-## Pseudo-Element resolution
-
-Pseudo-elements are a tricky section of the style system.
-Not all pseudo-elements are very common, and so some of them might want to skip the cascade.
-
-Servo has, as of right now, five [pseudo-elements][servo-pseudo-elements]:
-
- * [`::before`][mdn-pseudo-before] and [`::after`][mdn-pseudo-after].
- * [`::selection`][mdn-pseudo-selection]: This one is only partially implemented, and only works for text inputs and textareas as of right now.
- * `::-servo-details-summary`: This pseudo-element represents the `<summary>` of a `<details>` element.
- * `::-servo-details-content`: This pseudo-element represents the contents of a `<details>` element.
-
-Both `::-servo-details-*` pseudo-elements are private (i.e. they are only parsed from User-Agent stylesheets).
-
-Servo has three different ways of cascading a pseudo-element, which are defined in [`PseudoElementCascadeType`][pseudo-cascade-type]:
```

Comment:
can we remove the link definitions, if they are no longer referenced?

Reply:
Oh, thanks for catching that! I think I've removed all the unused links now.